CONTRIBUTING.md

```diff
@@ -87,69 +87,158 @@ You may need to install the dev_requirements for this
 
-4. Link IoT Hub to DPS - From DPS, click "Linked IoT Hub" and link the IoT Hub you just created.
+Note that if you stop the code with ctrl + C, the resources may not be deleted properly.
 
 #### Integration Test Environment Variables
 
 You can either manually set the environment variables or use the `pytest.ini.example` file in the root of the extension repo. To use that file, rename it to `pytest.ini`, open it and set the variables as indicated below.
 
 ```
     AZURE_TEST_RUN_LIVE=True
-    azext_iot_testrg="Resource Group that contains your IoT Hub"
-    azext_iot_testhub="IoT Hub Name"
-    azext_iot_testdps="IoT Hub DPS Name"
-    azext_iot_teststorageaccount="Storage Account Name"
-    azext_iot_teststoragecontainer="Blob container name"
+    azext_iot_testrg=
+    azext_iot_testhub=
+    azext_iot_testdps=
+    azext_iot_teststorageaccount=
+    azext_iot_teststoragecontainer=
+    azext_iot_central_app_id=
+    azext_iot_central_scope_id=
+    azext_iot_central_primarykey=
+    azext_iot_central_storage_cstring=
+    azext_iot_central_storage_container=
+    azext_dt_adx_cluster=
+    azext_dt_adx_database=
+    azext_dt_adx_rg=
+    azext_dt_ep_eventgrid_topic=
+    azext_dt_ep_servicebus_namespace=
+    azext_dt_ep_servicebus_policy=
+    azext_dt_ep_servicebus_topic=
+    azext_dt_ep_eventhub_namespace=
+    azext_dt_ep_eventhub_policy=
+    azext_dt_ep_eventhub_topic=
+    azext_dt_ep_eventhub_topic_consumer_group=
+    azext_dt_ep_rg=
+    azext_dt_region=
 ```
 
-`azext_iot_teststorageaccount` is the storage account used for running iothub storage tests. Optional variable, specify only when you want to run storage tests. During these tests, your hub will be assigned a System-Assigned AAD identity, and will be granted the role of "Storage Blob Data Contributor" on the storage account you provide. Both the hub's identity and the RBAC role will be removed once the test completes.
+To run almost all of the tests, only the `azext_iot_testrg` is needed.
+
+To run all tests, `azext_iot_testrg`, `azext_iot_central_app_id`, and `azext_iot_central_primarykey` are needed because the IoT Central Primary Key cannot be currently retrieved through the CLI.
+
+For all resources, if the environmental variable is not provided, a new instance will be created for the test run and deleted at the end of the test run.
+
+##### General Variables
+
+`az_iot_testrg` is the resource group that contains the IoT Hub and DPS instances or where all test resources are created. This is required, as it will be the default resource group if any other resource group variables are not provided.
+
+`azext_iot_testhub` is the test IoT Hub name. Optional variable, used for IoT Hub and DPS tests. If not provided, a new IoT Hub will be created for the test run (and deleted at the end of the test).
```

Comment:
`azext_iot_testhub` should probably be in an 'IoT Hub test variables' category

`azext_iot_testdps` should probably be in an 'IoT Device Provisioning Service' category